npm/vite-dev-server/src/makeCypressPlugin.ts

```diff
@@ -21,8 +21,8 @@ const INIT_FILEPATH = resolve(__dirname, '../client/initCypressTests.js')
 
 const HMR_DEPENDENCY_LOOKUP_MAX_ITERATION = 50
 
-function getSpecsSet (specs: Spec[]) {
-  return new Set<string>(specs.map((spec) => spec.absolute))
+function getSpecsPathsSet (specs: Spec[], supportFilePath) {
```

Comment:
Let's type it - I think the correct type is `supportFile?: string | null`, so you might need a null check, too.